Makefile

```diff
@@ -238,54 +101,71 @@ update_devworkspace_api:
 ### update_devworkspace_crds: pull latest devworkspace CRDs to ./devworkspace-crds. Note: pulls master branch
 update_devworkspace_crds:
 	./update_devworkspace_crds.sh $(DEVWORKSPACE_API_VERSION)
-
-
-### local: set up cluster for local development
-local: _print_vars _set_ctx _create_namespace _deploy_registry _set_registry_url _update_yamls _update_crds _update_controller_deps _reset_yamls _reset_ctx
-
-### generate: generates CRDs and Kubernetes code for custom resource
-generate:
-ifeq ($(shell operator-sdk version | cut -d , -f 1 | cut -d : -f 2 | cut -d \" -f 2),$(OPERATOR_SDK_VERSION))
-	operator-sdk generate k8s
-	operator-sdk generate crds
-	patch/patch_crds.sh
+###### End rules for dealing with devfile/api
+
+### test: Run tests
+ENVTEST_ASSETS_DIR = $(shell pwd)/testbin
+test: generate fmt vet manifests
+	mkdir -p $(ENVTEST_ASSETS_DIR)
+	test -f $(ENVTEST_ASSETS_DIR)/setup-envtest.sh || curl -sSLo $(ENVTEST_ASSETS_DIR)/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.6.3/hack/setup-envtest.sh
+	source $(ENVTEST_ASSETS_DIR)/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
+
+### manager: Build manager binary
+manager: generate fmt vet
+	go build -o bin/manager main.go
+
+### run: Run against the configured Kubernetes cluster in ~/.kube/config
+run: _generate_related_images_env
+	source $(RELATED_IMAGES_FILE)
+	go run ./main.go
+
+debug: _generate_related_images_env
+	source $(RELATED_IMAGES_FILE)
+	dlv debug --listen=:2345 --headless=true --api-version=2 ./main.go --
+
+### install: Install CRDs into a cluster
+install: manifests _kustomize _create_namespace
+	$(KUSTOMIZE) build config/crd | kubectl apply -f -
+
+#### uninstall: Uninstall CRDs from a cluster
+uninstall: manifests _kustomize
+	$(KUSTOMIZE) build config/crd | kubectl delete -f -
+
+### deploy: Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+deploy: _kustomize _create_namespace deploy_registry
+	mv config/devel/config.properties config/devel/config.properties.bak
+	mv config/devel/manager_image_patch.yaml config/devel/manager_image_patch.yaml.bak
+	envsubst < config/devel/config.properties.bak > config/devel/config.properties
+	envsubst < config/devel/manager_image_patch.yaml.bak > config/devel/manager_image_patch.yaml
+	$(KUSTOMIZE) build config/devel | kubectl apply -f - || true
+	mv config/devel/config.properties.bak config/devel/config.properties
+	mv config/devel/manager_image_patch.yaml.bak config/devel/manager_image_patch.yaml
+
+### restart: Restart devworkspace-controller deployment
+restart:
+	kubectl rollout restart -n $(NAMESPACE) deployment/devworkspace-controller-manager
+
+### uninstall_controller: Remove controller resources from the cluster
+uninstall_controller: _kustomize
```

Comment:
```
~/projects/devworkspace-operator update-sdk-1.x *3 !8 > make uninstall_controller                                                                            INT % 15:17:44
kustomize build config/devel | kubectl delete --ignore-not-found -f -
2020/11/02 15:17:45 got file 'controller.devfile.io_components.yaml', but '/home/sleshche/projects/devworkspace-operator/config/crd/bases/controller.devfile.io_components.yaml' must be a directory to be a root
Error: accumulating resources: recursed accumulation of path '/home/sleshche/projects/devworkspace-operator/config/default': accumulating resources: recursed accumulation of path '/home/sleshche/projects/devworkspace-operator/config/crd': accumulating resources: accumulating resources from 'bases/controller.devfile.io_components.yaml': empty item at spec/validation/openAPIV3Schema/properties/spec/properties/commands/items/properties/commandType/enum/ in object {map[apiVersion:apiextensions.k8s.io/v1beta1 kind:CustomResourceDefinition metadata:map[annotations:map[controller-gen.kubebuilder.io/version:v0.3.0] creationTimestamp:<nil> name:components.controller.devfile.io] spec:map[group:controller.devfile.io names:map[kind:Component listKind:ComponentList plural:components singular:component] scope:Namespaced subresources:map[status:map[]] validation:map[openAPIV3Schema:map[description:Component is the Schema for the components API properties:map[apiVersion:map[description:APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources type:string] kind:map[description:Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds type:string] metadata:map[type:object] spec:map[description:ComponentSpec defines the desired state of Component properties:map[commands:map[description:Commands from devfile, to be matched to components items:map[properties:map[apply:map[description:Command that consists in applying a given component definition, typically bound to a workspace event. 
 For example, when an `apply` command is bound to a `preStart` event, and references a `container` component, it will start the container as a K8S initContainer in the workspace POD, unless the component has its `dedicatedPod` field set to `true`. 
 When no `apply` command exist for a given component, it is assumed the component will be applied at workspace start by default. properties:map[attributes:map[additionalProperties:map[type:string] description:Optional map of free-form additional command attributes type:object] component:map[description:Describes component that will be applied type:string] group:map[description:Defines the group this command is part of properties:map[isDefault:map[description:Identifies the default command for a given group kind type:boolean] kind:map[description:Kind of group the command is part of enum:[build run test debug] type:string]] required:[kind] type:object] id:map[description:Mandatory identifier that allows referencing this command in composite commands, from a parent, or in events. type:string] label:map[description:Optional label that provides a label for this command to be used in Editor UI menus for example type:string]] required:[id] type:object] commandType:map[description:Type of workspace command enum:[Exec <nil>]]]]]]]]]]]]}
No resources found
```

Reply:
Tested locally and it seems to work fine; I'm not sure what the issue is.